versionhandler.py

```diff
@@ -2,7 +2,7 @@
 import re
 from typing import List, Optional, Tuple
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger("github-wikidata-bot")
```

Comment:
Just curious, what's this change about?

Reply:
yeah that should remain `logger = logging.getLogger(__name__)`

Reply:
We init the logger using 

```py
"loggers": {"github-wikidata-bot": {"handlers": handlers, "level": "INFO"}}
```

The logger name "github-wikidata-bot" must be used for every `logging.getLogger` call in order to yield the desired configuration.

`__name__` differs for every file. Reference: https://docs.python.org/3.11/library/__main__.html#name-main

Reply:
normally you define the logger config for your whole package/module. Could you mode the files into a `github_wikidata_bot` module and change the logger config to snake case?

Reply:
Done.